### DISCUSSION TITLE
Can't access community exchange

### DISCUSSION THREAD

**Discussion by BennettSchwartz** [community/community#142097] [GitHub Education]
### Hi everyone,

When I was trying to open the community exchange link, I got this:
![CleanShot 2024-10-20 at 21 25 11@2x](https://github.com/user-attachments/assets/30c910cc-0241-445f-972d-7a6464da2798)

I've tried to restart, clear cache, sign in and out, and use a different browser.

**Comment by kilva245** [comment]
hi link please

**Reply by BennettSchwartz** [reply]
Hey! Thanks for your quick response :)

The link is: https://education.github.com/globalcampus/exchange
